telemetry/request.go

```diff
@@ -30,7 +30,7 @@ type FunctionLogMessage struct {
 	Attributes map[string]interface{} `json:"attributes"`
 }
 
-func NewFunctionLogMessage(timestamp int64, requestId string, message string) FunctionLogMessage {
+func NewFunctionLogMessage(timestamp int64, requestId, traceId, message string) FunctionLogMessage {
```

Comment:
Do we want to remove the type hinting for requestId and add a no-type-hint traceId term?

Reply:
This is a Goism. `foo, bar, baz string` is equivalent to `foo string, bar string, baz string`.